frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -48,7 +51,10 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
             hasDiskType = DISK_TYPES[state.diskType].property === disk.property;
           }
 
-          if (isValidSize && hasDiskType) {
+          const hasDeviceType: boolean = state.deviceType.includes(
+            deviceTypeDropdownItems[disk.type.toUpperCase()],
```

Comment:
Now when we are storing the values , we you can just go with `disk.type`
```suggestion
          const hasDeviceType: boolean = state.deviceType.includes(disk.type)
```

Reply:
disk.type has  values that are populated using  LSO CR, The values are 'part' or 'disk' (small case) and state.deviceType has ['Disk', 'Part'],

Reply:
Ah! the object is not helpful here, its for dropdown only.
We shall use this one in default state and then it should be fine.
https://github.com/openshift/console/blob/ef18881cf0cf1b640a631c9ddc6f2f979309ca67/frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/types.ts#L12-L15

Reply:
Okay, I see the dropdown options are getting set only.
In that case this is good in place for now.